docs/Configuration/Configuration.md

```diff
@@ -0,0 +1,149 @@
+
+![Image of Logic Console sidebar menu](./assets/image001.jpg)
+
+After doing that you will be presented with the `Logic Document Generation Configuration` module.
+
+![Image of Logic Console Document Generation Configuration module](./assets/image002.jpg)
+
+
+## Creating a Configuration environment
+
+To create a Document Generation Configuration click on the `ADD CONFIGURATION` button at the top right corner of the screen.
+
+You will be then presented with the following screen:
+
+![Image of Logic Console Create Document Generation Configuration screen](./assets/image003.jpg)
+
+The screen is divided in two sections, the `Configuration Settings` and the `Master Storage Settings`, we will explain them below in detail.
+
+### Configuration Settings
+
+![Image of Logic Console Create Document Generation Configuration - Settings](./assets/image004.jpg)
+
+- Configuration name:  
+_(Required)_ The configuration name, any non-empty string is valid.
+
+- ASPOSE license switch:  
+_(Required)_ In order to use `Logic Document Generation` your organization needs to have a valid ASPOSE license. Toggle this switch on to acknowledge that.
+
+- Levels:
+_(Optional)_ `Logic Document Generation` allows you to consume templates stored in multiple storages, it uses a hierarchical organization where each _**level**_ down the hierarchy is able to override a template found on its parent _**level**_.  
+By default this screen is pre-populated with two levels (Customer and Department), feel free to modify this to suit your needs.  
+A configuration environment can have zero, one or multiple levels.  
+We will see later how to configure entries at different levels of the hierarchy.
+
+### Master Storage Settings
+
+- Template provider:  
+_(Required)_ The template storage provider where the master templates are stored.  
+`Logic Document Generation` currently supports two storage providers: `SharePoint 365 Online` and `Azure Blob Storage`.
+
+#### SharePoint settings
+![Image of Logic Console Create Document Generation Configuration - SharePoint Settings](./assets/image005.jpg)
+
+`Logic Document Generation` requires a valid SharePoint Client Credential and a Group name to retrieve templates from it.  
+**The Client Credential needs to be granted permissions to read documents and the SharePoint instance Group collection.**
+
+- Client Id:  
+_(Required)_ The Client Id from your SharePoint Client Credentials.
+
+- Tenant Id:  
+_(Required)_ The Tenant Id from your SharePoint Client Credentials.
+
+- Group Name:  
+_(Required)_ The SharePoint group name where the templates are stored.
+
+- Client Secret:  
+_(Required)_ The Client Secret from your SharePoint Client Credentials.
+
+#### Azure Blob Storage settings
+![Image of Logic Console Create Document Generation Configuration - Azure Blob Storage Settings](./assets/image006.jpg)
+
```

Comment:
How does someone create these? I assume create a blob storage, add a service principal, copy these details from here to here ...

Reply:
I will address this in a separate PR.